tests/Jellyfin.Naming.Tests/TV/DailyEpisodeTests.cs

```diff
@@ -23,12 +23,13 @@ public void Test(string path, string seriesName, int? year, int? month, int? day
             var result = new EpisodeResolver(options)
                 .Resolve(path, false);
 
-            Assert.Null(result.SeasonNumber);
-            Assert.Null(result.EpisodeNumber);
-            Assert.Equal(year, result.Year);
-            Assert.Equal(month, result.Month);
-            Assert.Equal(day, result.Day);
-            Assert.Equal(seriesName, result.SeriesName, true);
+            Assert.NotNull(result);
+            Assert.Null(result!.SeasonNumber);
```

Comment:
Looks like xunit 3.0 fixes this (whenever that releases...)

Reply:
Yeah, I'd seen that they weren't planning further 2.x releases, but it doesn't look like there's a target date for 3.0, so stuck with the workarounds until then.